docs/FAQ.md

```diff
@@ -355,18 +355,18 @@ for the RAPIDS plugin.
 
 ### Are table layout formats supported?
 
-Yes, there is GPU support for [Delta Lake](./additional-functionality/delta-lake-support.md) and
-[Apache Iceberg](./additional-functionality/iceberg-support.md). See the additional support
-documentation for specifics on the operations supported for these formats.
+Yes, there is GPU support for Delta Lake read 
+and [Apache Iceberg 0.13 read](./additional-functionality/iceberg-support.md).  
+Write operations will fall back to the CPU.
```

Comment:
This removed the link to the Delta Lake support section.  This section should have been left untouched from what was already in branch-22.12.

Reply:
@nvliyuan can you help address this comment?

Reply:
sure, already draft a [pr](https://github.com/NVIDIA/spark-rapids/pull/7346)